projects/1.16/assets/turtlemancy/turtlemancy/lang/zh_cn.json

```
@@ -0,0 +1,143 @@
+	"item.turtlemancy.suspicious_stew_night_vision": "迷之炖菜",
+	
+	"item.turtlemancy.turtle_essence": "海龟精华",
+	"item.turtlemancy.water_creature_essence": "水生生物精华",
+	"item.turtlemancy.enderman_essence": "末影人精华",
+	"item.turtlemancy.animal_essence": "动物精华",
+	"item.turtlemancy.human_essence": "人类精华",
+	"item.turtlemancy.undead_essence": "不死精华",
+	"item.turtlemancy.arthropod_essence": "节肢生物精华",
+	
+	"item.turtlemancy.tooltip": "按住SHIFT键获取更多信息",
+	
+	"item.turtlemancy.turtlemancer_basic_wand": "\u00A72基础海龟巫师魔杖",
+	"item.turtlemancy.turtlemancer_basic_wand.tooltip.line1": "最基础的\u00A72海龟巫师\u00A7r魔杖。只能施放1级法术。",
+	"item.turtlemancy.turtlemancer_wand": "\u00A72海龟巫师魔杖",
+	"item.turtlemancy.turtlemancer_wand.tooltip.line1": "普通的\u00A72海龟巫师\u00A7r魔杖。可以施放1级法术和2级法术。",
+	"item.turtlemancy.turtlemancer_advanced_wand": "\u00A72进阶海龟巫师魔杖",
+	"item.turtlemancy.turtlemancer_advanced_wand.tooltip.line1": "最强大的\u00A72海龟巫师\u00A7r魔杖。能够施放所有法术。",
+	
+	"item.turtlemancy.turtle_helmet": "海龟巫师头盔",
+	"item.turtlemancy.turtle_chestplate": "海龟巫师胸甲",
+	"item.turtlemancy.turtle_leggings": "海龟巫师护腿",
+	"item.turtlemancy.turtle_boots": "海龟巫师靴子",
+	
+	"item.turtlemancy.magmatic_turtle_helmet": "狱炎海龟巫师头盔",
+	"item.turtlemancy.magmatic_turtle_chestplate": "狱炎海龟巫师胸甲",
+	"item.turtlemancy.magmatic_turtle_leggings": "狱炎海龟巫师护腿",
+	"item.turtlemancy.magmatic_turtle_boots": "狱炎海龟巫师靴子",
+	
+	"item.turtlemancy.dark_turtlemancer_wand": "\u00A75邪恶海龟巫师魔杖",
+	"item.turtlemancy.dark_turtlemancer_wand.tooltip.line1": "普通的\u00A75邪恶海龟巫师\u00A7r魔杖。几乎能够施放所有的1级法术和2级法术，但也能释放一些邪恶法术。",
+	"item.turtlemancy.dark_turtlemancer_advanced_wand": "\u00A75进阶邪恶海龟巫师魔杖",
+	"item.turtlemancy.dark_turtlemancer_advanced_wand.tooltip.line1": "最强大的\u00A7r邪恶海龟巫师\u00A7r魔杖。能够施放所有的邪恶法术和大多数常规法术。",
+	
+	"item.turtlemancy.bewitched_shell_necklace": "\u00A75魔法贝壳项链",
+	"item.turtlemancy.heart_of_the_sea_bracelet": "\u00A79海洋手镯",
+	"item.turtlemancy.tidal_ring": "潮汐戒指",
+	"item.turtlemancy.tephra_ring": "火山戒指",
+	"item.turtlemancy.nautilus_belt": "鹦鹉螺腰带",
+	"item.turtlemancy.deep_blue_charm": "深蓝护符",
+	"item.turtlemancy.magma_heart": "狱炎之心",
+	
+	"item.turtlemancy.spawn_egg_magma_turtle": "狱炎海龟刷怪蛋",
+	
+	"item.turtlemancy.shell_of_encapsulation": "\u00A72封装贝壳",
+	"item.turtlemancy.filled_shell_of_encapsulation": "\u00A72已填充的封装贝壳",
+	"item.turtlemancy.filled_shell_of_encapsulation.desc": "填充实体：",
+	
+	"block.turtlemancy.oyster_sand": "埋有牡蛎的沙子",
+	"block.turtlemancy.pearl_block": "珍珠块",
+	"block.turtlemancy.turtle_cauldron": "海龟巫师坩埚",
+	"block.turtlemancy.turtle_altar": "海龟巫师祭坛",
+	"block.turtlemancy.mine_turtle": "海龟地雷",
+	"block.turtlemancy.magma_turtle_egg": "狱炎海龟蛋",
+	"block.turtlemancy.turtle_door": "镀金鳞甲门",
+	"block.turtlemancy.improved_turtle_door": "改良鳞甲门",
+	
+	"entity.turtlemancy.magma_turtle": "狱炎海龟",
+	"entity.turtlemancy.turtle_boat": "海龟船",
+	
+	"container.turtlemancy.turtle_altar": "海龟祭坛",
+	
+	"enchantment.turtlemancy.turtles_aqua_affinity": "海龟水下速掘",
+	
+	"itemGroup.turtlemancytab": "魔之海龟",
```

```suggestion
	"itemGroup.turtlemancytab": "海龟魔法",
```
仅建议，-mancy词尾指代魔法或某类魔法，如pyromancy、necromancy、golemancy
如修改则全文件通查